R/crossValidate.R

```diff
@@ -82,3 +170,6 @@ summary.crossvalidation <- function(object, ...) {
   s$Catchment <- sapply(object, function(x) x$name.Catchment)
   s
 }
+
+#' @import utils
+utils::globalVariables(c("%dopar%"))
```

Comment:
dopar is defined in the foreach package. I assume it is added as globalVariables because there's no way of using the namespace prefix on infix functions?

Reply:
I added it as a global variable due to the nature of the error R CMD check gave:

`crossValidate: no visible global function definition for ‘%dopar%’`

[https://github.com/josephguillaume/hydromad/runs/905472912?check_suite_focus=true](https://github.com/josephguillaume/hydromad/runs/905472912?check_suite_focus=true)